app/views/category/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= t(".heading") %></h1>
+
+<p><%= t(".page_description") %></p>
+
+<p><%= link_to I18n.t(".navigation.helpers.back"), root_path %></p>
+
+<ul>
+<% categories.each do |category| %>
+  <li><%= link_to category.name, category_path(category.id) %></li>
```

Comment:
I think you can just pass the category into `category_path` without having to specify the id.

```ruby
link_to category.name, category_path(category)
```